services/appInsight_ui.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react";
```

Comment:
I think the convention for this file would be to put it in `/content/appInsights.ts`